src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -59,12 +62,32 @@ public static Set<Tag> getTagSet(String... strings) {
     }
 
     /**
-     * Returns a tutorial set containing the list of strings given.
+     * Returns a tutorial set containing the list of strings given and all are defaulted to true.
```

Comment:
Should be defaulted to not taken place yet.

Reply:
Thanks for catching this! Amended.